packages/core/ai_content_pipeline/ai_content_pipeline/pipeline/step_executors/audio_steps.py

```
@@ -0,0 +1,141 @@
+        """
+        Initialize executor with generator.
+
+        Args:
+            generator: ReplicateMultiTalkGenerator instance
+        """
+        self.generator = generator
+
+    def execute(
+        self,
+        step,
+        input_data: Any,
```

Ruff (0.14.10)

Unused method argument: `input_data`

(ARG002)